openmc/data/data.py

```diff
@@ -223,19 +223,19 @@ def atomic_mass(isotope):
     """
     if not _ATOMIC_MASS:
 
-        # Load data from AME2016 file
-        mass_file = os.path.join(os.path.dirname(__file__), 'mass16.txt')
+        # Load data from AME2020 file, Note format change to AME2016
```

Comment:
Comment won't really make sense to someone who isn't aware that we used to use 2016 data, so I'd suggest just removing that bit
```suggestion
        # Load data from AME2020 file
```